terragrunt/aws/api/outputs.tf

```
@@ -0,0 +1,4 @@
+output "function_url" {
+  description = "The URL of the Lambda function"
+  value       = aws_lambda_function_url.api.function_url
```

I'd recommend marking this as `sensitive` just to make it harder for people to figure out the function URL:
```suggestion
  value       = aws_lambda_function_url.api.function_url
  sensitive   = true
```